### PR TITLE
Slack to Discord migration

### PR DESCRIPTION
**Pull request by jtc**
Since UQCS has migrated from Slack to Discord, this pull request updates any Slack references on the website to now point to Discord.

Note: historical references on the CodeJam event pages have been left as is.

Also hyperlinked the previous plain text on the Community page, as it seemed to make sense.

Updates build and look fine locally, except the nav/footer social icon seems a little mis-aligned, since most other icons are square where the Discord icon is a little taller. Thinking about what to do there but welcome comments.

### CODE REVIEW

**Comment by jtc** [comment]
Here's the situation with the nav/footer social icons:
Screenshot of UQCS website footer social icons with annotation to show alignment of Discord icon
Since most of the icons are square, while the Discord icon is _technically_ centred with the rest, it looks a little higher since the "square part" doesn't align with the other squares, due to its triangular bottom.

In Font Awesome v6, the [Discord icon will change](https://fontawesome.com/v6.0/icons/discord) in-line with [Discord's logo update](https://support.discord.com/hc/en-us/articles/1500009438682-A-Fresh-New-Look-to-Celebrate-Our-6th-Birthday), removing the square-ish "chat bubble" and leaving just the core smiley icon. This alone will solve the issue.

In the meantime, and perhaps anyway, there are a few options if the current look is deemed unsatisfactory:
1. Play with the alignment manually to move the Discord icon down relative to the others, to make it "look" more in-line. (this is probably beyond my limited skills so would need some help)
2. Change to the default [fa-facebook](https://fontawesome.com/v5.13/icons/facebook), [fa-twitter](https://fontawesome.com/v5.13/icons/twitter) and [fa-github](https://fontawesome.com/v5.13/icons/github) icons so we're not aiming for any square consistency anyway.
Screenshot of UQCS website footer social icons if using all default/official icons
3. As per (2) but also move to [fa-facebook-f](https://fontawesome.com/v5.13/icons/facebook-f) and [fa-linkedin-in](https://fontawesome.com/v5.13/icons/linkedin-in) to move even further away from enclosing circle/squares.
Screenshot of UQCS website footer social icons if using all available options without enclosing shapes

I'm personally in favour of doing (2) or (3) anyway as Slack (rip) / FAv6 Discord and YouTube don't have square options, and I'm not a big fan of the look of [fa-github-square](https://fontawesome.com/v5.13/icons/github-square) especially at small sizes.

(2) is my personal preference since it's the "official" logos but up to you 😃

It may then be worth looking at the Facebook and GitHub icons on the [Connect page](http://uqcs.org/connect/) too.

Thoughts?

**Comment by TomCranitch** [comment]
Thanks for all these fixes, it is much appreciated. My preference is probably for (2) as well but I'm not sure if @JamesDearlove or @kentonlam have any strong opinions otherwise.

File: content/community/_index.html [hunk 1]
```diff
@@ -13,8 +13,8 @@ <h1 class="title is-3">UQCS Community</h1>
                 In the past, we've run Games, Jackbox, Trivia and Movie Nights, BBQs, study sessions, end-of-semester
                 drinks, and workshops; we also frequently collaborate with other tech and related interests societies,
                 both in and out of UQ.</p>
-            <p>We have an extremely active Slack community consisting of thousands of members, comprised of current 
-                students, alumni, and other people in the Brisbane tech community.</p>
+            <p>We have an extremely active <a href="https://discord.uqcs.org/">Discord community</a> consisting of thousands
```
Comment (TomCranitch) [inline content/community/_index.html:16]: We can no longer way we have thousands of students. The easiest fix might just be to change that to say hundreds

Comment (jtc) [reply content/community/_index.html]: Good point, have committed 7ab3b4e to fix this.

Comment (jdcaperon) [reply content/community/_index.html]: Could also just not mention the count until it gets to the point of it being worth mentioning again

Comment (jtc) [reply content/community/_index.html]: Done 5a503c5

**Comment by jtc** [comment]
> 2. Change to the default [fa-facebook](https://fontawesome.com/v5.13/icons/facebook), [fa-twitter](https://fontawesome.com/v5.13/icons/twitter) and [fa-github](https://fontawesome.com/v5.13/icons/github) icons so we're not aiming for any square consistency anyway.
>    Screenshot of UQCS website footer social icons if using all default/official icons
>
> ... 
> It may then be worth looking at the Facebook and GitHub icons on the [Connect page](http://uqcs.org/connect/) too.


173f5f1 implements this but easy change if preference for other options.

Otherwise, ready for further review/merge.

**Review by jdcaperon — APPROVED** [review]
LGTM

Maybe we could have a historical section on the slack one day, a list of all the emojis or something.

**Comment by nicklambourne** [comment]
> Maybe we could have a historical section on the slack one day, a list of all the emojis or something.

Big fan of this idea, I'll see if one of the emoji nerds has a recent archive of them all we can use.

**Review by nicklambourne — APPROVED** [review]
LGTM

**Pull request merged**